packages/reports/addon/templates/components/navi-column-config/base.hbs

```diff
@@ -1,32 +1,49 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-<div class="navi-request-column-config" ...attributes>
-  <div class="navi-request-column-config__header">
-    <span class="navi-request-column-config__header-label">Column Configuration</span>
-    <span class="navi-request-column-config__header-icons">
-      <NaviIcon @icon="star-o" class="navi-request-column-config__star-icon" role="button"/>
-      <NaviIcon @icon="times" class="navi-request-column-config__exit-icon" role="button" {{action this.onClose}}/>
-    </span>
-  </div>
-  <div class="navi-request-column-config__body">
-    <div class="navi-request-column-config__option navi-request-column-config__column-name">
-      <label for="columnName" class="navi-request-column-config__column-name-label navi-request-column-config__option-label">Column Name</label>
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="navi-column-config-base" ...attributes>
+  <div class="navi-column-config-base__body">
+  {{#if (or (eq @column.type "metric") (eq @column.type "dimension"))}}
+    <div class="navi-column-config-base__option navi-column-config-base-type">
+      <div class="navi-column-config-base__header">
+        <label for="{{this.classId}}-base-type-input" class="navi-column-config-base__base-type-label navi-column-config-base__option-label">Base {{capitalize @column.type}}</label>
+        <span class="navi-column-config-base__header-icons">
+          {{!-- TODO: Favorites are not supported yet --}}
+          {{!-- <NaviIcon @icon="star-o" class="navi-column-config-base__star-icon" role="button" /> --}}
+          <NaviIcon @icon="clone" class="navi-column-config-base__clone-icon" role="button" {{on "click" @cloneColumn}} />
+          <NaviIcon 
+            class="navi-column-config-base__filter-icon {{if column.isFiltered "navi-column-config-base__filter-icon--active"}}"
+            @icon="filter" 
+            role="button" 
+            {{on "click" @toggleColumnFilter}} 
+           />
+        </span>
+      </div>
+      <Input
+        class="navi-column-config-base__base-type-input navi-column-config-base__option-input"
+        id="{{this.classId}}-base-type-input"
+        @disabled={{true}}
```

Comment:
A column on this line would be helpful `Possibly enable and allow changing base metric here in the future`